src/components/CommandBar/CommandBar.tsx

```diff
@@ -149,6 +151,7 @@ export class CommandBar extends React.Component<ICommandBarProps, ICommandBarSta
                          data-command-key={ index }
                          aria-haspopup={ !!(item.items && item.items.length) }
                          role='menuitem'
+                         aria-label={ `${ item.name }. ${ index === 0 ? this.props.itemNavigationScreenReaderText : '' }` }
```

Comment:
Followed up with Chris, thanks! Switched to use ariaLabel now for all buttons. Sorry wasn't aware of the use ariaLabel in other controls.